packages/polyserve/src/start_server.ts

```diff
@@ -555,3 +558,13 @@ async function tryStartWithPort(
     });
   });
 }
+
+// TODO(usergenic): Something changed in the typings of net.Server.address() in that
```

Comment:
This was https://github.com/DefinitelyTyped/DefinitelyTyped/pull/25597. It is apparently correct that net.Server can return a string, but only when it's over pipes or Unix domain sockets, according to https://nodejs.org/api/net.html#net_server_address. I wonder if it's possible start an HTTP server over either of those. If not maybe the HTTP Server class type should be updated to override address() to say that it can only return the info object.

Reply:
@aomarks I have seen HTTP servers connected via a Windows Pipe on Microsoft azure web apps.